tasks/systemd_timers.yml

```
@@ -0,0 +1,53 @@
+---
+
+- name: 'Test apt-daily timer'
+  ansible.builtin.command:
+    cmd: 'systemd-analyze calendar "{{ unattended_apt_daily_oncalendar }}"'
+  register: 'apt_daily'
+  changed_when: 'apt_daily.rc != 0'
+  check_mode: 'no'
+
+- name: 'Test apt-daily-upgrade timer'
+  ansible.builtin.command:
+    cmd: 'systemd-analyze calendar "{{ unattended_apt_daily_upgrade_oncalendar }}"'
+  register: 'apt_daily_upgrade'
+  changed_when: 'apt_daily_upgrade.rc != 0'
+  check_mode: 'no'
+
+- name: 'Ensure apt-daily.timer.d exist'
+  ansible.builtin.file:
+    path: /etc/systemd/system/apt-daily.timer.d
+    mode: '0755'
+    state: directory
+  register: 'apt_daily_timer_d'
+
+- name: 'Ensure apt-daily-upgrade.timer.d'
+  ansible.builtin.file:
+    path: /etc/systemd/system/apt-daily-upgrade.timer.d
+    mode: '0755'
+    state: directory
+  register: 'apt_daily_upgrade_timer_d'
+
+- name: 'Deploy apt-daily timer'
+  ansible.builtin.template:
+    src: 'apt_daily_override.conf.j2'
+    dest: '/etc/systemd/system/apt-daily.timer.d/override.conf'
+    mode: 0644
```

Quotes would make sense here as well:

```suggestion
    mode: '0644'
```